frontend_streamlit/streamlit_app.py

```diff
@@ -71,12 +82,15 @@ def app():
     date_range = st.sidebar.slider('Select a range of dates', 2015, 2022, (2016, 2019))
     clicked = st.sidebar.button('Submit')
     if clicked and user_text and nb_articles > 0:
-        st.write("You entered: ", user_text)
+        st.markdown(f'<h2 style="color:#1126ad;font-size:30px;">You\'ve entered: <br><em>\"{user_text}\"</em></h1>',
```

Comment:
Je trouve ça un peu gros (et difficilement lisible vu la couleur) en terme de rendu, je trouve que le texte en petit et blanc se voit mieux

Reply:
<img width="932" alt="image" src="https://user-images.githubusercontent.com/66633561/198966759-38118214-bc31-4b97-9e4a-81c5e75c1b60.png">

Je pense que for some reason ton app n'applique pas le config.toml, mais du coup voici comment ça rend. Et c'est pareil sur l'app deployée du coup